### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by krail**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [x] **typo fix**
- [ ] **metadata update**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -314,7 +314,10 @@ module.exports = {
       {
         test: /\.lazy\.css$/i,
         use: [
-          { loader: 'style-loader', options: { injectType: 'lazyStyleTag' } },
+          {
+            loader: 'style-loader',
+            options: { injectType: 'lazySingletonStyleTag' },
```
Comment (anikethsaha) [inline README.md]: why not `lazyStyleTag` ?

Comment (krail) [reply README.md]: This section is about `lazySingletonStyleTag`.

Comment (anikethsaha) [reply README.md]: my bad  :)

**Comment by anikethsaha** [comment]
Please check your linting errors

**Comment by anikethsaha** [comment]
you have error in your commit message

try something like this 

`fix: typos`

**Pull request merged**